src/Sylius/Bundle/AdminBundle/Resources/views/ProductVariant/Field/inventory.html.twig

```diff
@@ -1,10 +1,10 @@
 {% if data.isTracked %}
 <div class="ui {{ data.onHand > 0 ? 'green' : 'red' }} icon label">
     <i class="cube icon"></i>
-    {{ data.onHand }} {{ 'sylius.ui.available_on_hand'|trans }}
+    <span id="sylius-{{ data.id }}-on-hand-quantity">{{ data.onHand }}</span> {{ 'sylius.ui.available_on_hand'|trans }}
```

Comment:
`id="sylius-{{ data.id }}-on-hand-quantity"` -> `class="sylius-on-hand-quantity" data-product-variant-id="{{ data.id }}"`? These non-semantic IDs made only for Behat looks quite bad to me :/